hack/dind-cluster.sh

```diff
@@ -44,59 +43,58 @@
 # command would ensure configuration was stored at
 # /tmp/openshift-dind/cluster/my-cluster:
 #
-#    OS_INSTANCE_PREFIX=my-cluster hack/dind-cluster.sh [command]
+#    OPENSHIFT_INSTANCE_PREFIX=my-cluster hack/dind-cluster.sh [command]
 #
 # It is also possible to specify an entirely different configuration path:
 #
-#    OS_DIND_CONFIG_ROOT=[path] hack/dind-cluster.sh [command]
+#    OPENSHIFT_CONFIG_ROOT=[path] hack/dind-cluster.sh [command]
 #
-# Running Tests
-# -------------
-#
-# This script includes a shortcut for running the networking e2e
-# tests.  The test-net-e2e command will run the extended networking
-# tests against an already running dind cluster:
-#
-#     hack/dind-cluster.sh test-net-e2e
```

Comment:
The tests need to be run by test/extended/networking.sh as per the updated CONTRIBUTING.adoc.